scripts/validate-marketplace-okd.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+# Copyright 2024 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+set -x
+
+if [ -z "${TIMEOUT}" ]; then
+    echo "Please set TIMEOUT"; exit 1
+fi
+
+OPERATORS="cert-manager metallb-operator"
```

Comment:
On OKD we install the nmstate operator from source [1] because the latest version available in community catalog is too old [2]

[1] https://github.com/openstack-k8s-operators/install_yamls/blob/main/scripts/gen-olm-nmstate-okd.sh#L41-L46
[2] https://github.com/k8s-operatorhub/community-operators/tree/main/operators/kubernetes-nmstate-operator